src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -201,10 +201,11 @@
                             (neg? change-pct-24hour) :negative
                             :else                    :empty)
      :customization-color color
-     :values              {:crypto-value      crypto-value
-                           :fiat-value        fiat-value
-                           :fiat-change       formatted-token-price
-                           :percentage-change percentage-change}}))
+     :values              {:crypto-value           crypto-value
+                           :fiat-value             fiat-value
+                           :fiat-unformatted-value fiat-unformatted-value
```

Comment:
We need the unformatted fiat value without the currency symbol and less than "<" sign to sort properly